lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -207,13 +227,18 @@ private void migrateIdentity(
             ReportUserIdentityItem reportItem, PageSummary pageSummary, String batchId) {
         // Skip any non P2 identities
         if (!Vot.P2.name().equals(reportItem.getIdentity())) {
-            // Send skipped audit event
             LOGGER.info(
                     annotateLog(
                             "Skipping migration - not a P2 identity",
                             reportItem,
                             batchId,
                             pageSummary));
+            auditService.sendAuditEvent(
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_EVCS_MIGRATION_SKIPPED,
+                            batchId,
+                            new AuditEventUser(reportItem.getHashUserId(), "", "", ""),
+                            new AuditExtensionsEvcsSkippedMigration(batchId, "already_migrated")));
```

Comment:
I think the reason here is wrong - this is for non P2 identities. So somthing like `not_p2_identity` perhaps